client/components/download-link/renderers/docx-renderer.js

```diff
@@ -119,6 +121,104 @@ export default (application, sections, values) => {
     return doc;
   };
 
+  const renderTable = (doc, node) => {
+    const tableToMatrix = table => {
```

Comment:
I'd be tempted to pull this function up a level if it doesn't need the `doc` or `node` variables in scope (and I don't think it does). Purely to make the code more readable.

Reply:
right you are